sources/packages/web/src/services/http/Institution.ts

```diff
@@ -115,4 +115,23 @@ export class InstitutionApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async checkIfExist(guid: string, headers: any): Promise<boolean> {
+    try {
+      const resp: AxiosResponse<void> = await this.apiClient.head(
+        "institution/" + guid,
```

Comment:
can we use **Template literals** for the sake of consistency, as we follow  **Template literals** in all other Axios calls

Reply:
Template literals has been used as discussed.